### PR TITLE
chore: fix playground build

### PR DESCRIPTION
**Pull request by lifeiscontent**
@edmundhung I'm hoping this will fix the issues we're seeing in playwright

### CODE REVIEW

File: guide/package.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
 		"eslint": "^8.35.0",
 		"tailwindcss": "^3.4.0",
 		"typescript": "^5.3.3",
-		"wrangler": "^3.22.1"
+		"wrangler": "^3.28.2"
```
Comment (lifeiscontent) [inline guide/package.json:36]: running `pnpm dedupe` said these versions were the correct requested versions from `@remix-run/dev`

File: playground/tailwind.config.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default {
+	content: ['./app/**/*.{js,jsx,ts,tsx}'],
+	theme: {
+		extend: {},
+	},
+	plugins: [
+		form,
+		plugin(function ({ addVariant }) {
+			addVariant('touched', '&[data-conform-touched="true"]');
```
Comment (lifeiscontent) [inline playground/tailwind.config.ts:13]: @edmundhung is this an artifact from previous versions of conform? maybe we can remove?

Comment (edmundhung) [reply playground/tailwind.config.ts:13]: We can remove it. It's a legacy thing pre v1.

**Review by edmundhung — DISMISSED** [review]
Thanks! I think the CI issue should be already resolved with #614, but it's great to update the playground app anyway.

**Review by edmundhung — APPROVED** [review]
Thanks! :)

**Pull request merged**